include/sprite.h

```diff
@@ -205,6 +205,13 @@ typedef struct {
     /* 0x06 */ u8 variant;
 } TileInfo;
 
+// NOTE: (Only?) used by by src/game/bosses/final_intro.c
```

Comment:
If that's the case why not just define it in final intro?

Reply:
I'm not sure because I haven't checked the whole code for similar uses, that's why there's a questionmark! ^^

Reply:
Ok :)